lib/mcts/playout.rb

```diff
@@ -1,22 +0,0 @@
-module MCTS
-  class Playout
-
-    attr_reader :game_state
-
-    def initialize(game_state)
-      @game_state = game_state.dup
-    end
-
-    def play
-      my_color = @game_state.last_turn_color
-      playout
-      @game_state.won?(my_color)
-    end
-
-    def playout
-      until @game_state.finished?
-        @game_state.set_move @game_state.generate_move
-      end
-    end
```

Comment:
I see that your version is simpler, I'd still vote to keep the `Playout` object as itself alive. It's in the spirit of the Single Responsibility Principle (the node shouldn't know how to do this imo) + for benchmarking a playout can be benchmarked independently without needing to setup a node or tree. See `playout_help` :)

```
def playout_object_for(size)
  MCTS::Playout.new(Rubykon::GameState.new Rubykon::Game.new(size))
end
```

Reply:
Ah, I see - yeah this makes sense to me now.